packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -370,6 +380,16 @@ class NewDisk extends Component {
             </ActionButton>
           </span>
         </fieldset>
+        {!this._isSrOnSameHost() && (
+          <div>
+            <span className='text-danger'>
+              <Icon icon='alarm' />{' '}
+              {_('srNotOnSameHost', {
+                host: definedGet(() => hosts[requiredContainer].name_label),
```

Comment:
You need to check for `undefined` before otherwise the message will look strange.

Should we still show a warning if the user doesn't have permissions on the host, though? We decided not to show any for the "current VM state" case but here, the user might be adding a disk that will break the VM.